specs/select/huge_arrays.toml

```diff
@@ -1,12 +1,9 @@
 [setup]
 statement_files = ["../sql/huge_arrays.sql"]
 
-    [[setup.data_files]]
-    # Data must be generated first using ./generators/huge_arrays.sh > specs/data/huge_arrays.json
+    [[setup.data_cmds]]
     target = "huge_arrays"
-    source = "../data/huge_arrays.json"
-    bulk_size = 500
```

Comment:
Is it okay to drop the `bulk_size` property? I suspect this may be related to using less memory as otherwise the request size of the inserts may be too big.

Reply:
restored it - somehow I thought that this config is not related to data-gen case, thx for spotting

Reply:
It's also used in the data generation case: https://github.com/mfussenegger/cr8/blob/965074b6878e859551b5b7872008a13f75291b22/cr8/run_spec.py#L137